tsconfig.json

```diff
@@ -1,15 +1,14 @@
 {
 	"compilerOptions": {
 		"module": "commonjs",
-		"target": "ES2020",
+		"target": "ES2022",
 		"outDir": "out",
 		"lib": [
-			"ES2020"
+			"ES2022"
 		],
 		"sourceMap": true,
 		"rootDir": "src",
-		"skipLibCheck": true,
-		"strict": true   /* enable all strict type-checking options */
```

Comment:
what is the reason to remove the turn `strict` to `false`?

Reply:
good point, I have forgot to mention that and discuss further the best approach..

it is mainly because we wanted to reuse same functions as in LSP without changing them, problem is that LSP project is not using strict and same tsconfig setup as DAP extension which causes problems.. so we were considering in the first iteration to reuse same functions to stay consistent for utils we are using across projects and in the next iteration we wanted to unify tsconfigs for both projects and fix utils functions which could be done in another issue/PR (already opened and selected for dev https://issues.redhat.com/browse/FUSETOOLS2-2256)

so we wrere considering options like what is better..
1) for this PR revert strict change, copy functions and modify them to be compliant but in that case different from LSP ones
2) remove strict for some time then fusetools2-2256 will be resolved and reuse same functions for now

in the end probably better to let strict there as is and revise everything for both projects in another mentioned issue 🤷‍♂️.. I can see Filip already reverted tsconfig change of strict option

Reply:
basically problem is that we need to 
1) unify tsconfigs across our extensions projects 
2) finally provide just one shared test utils we can share across projects to prevent these copy&paste and hard maintain issues..

Reply:
both solution seems reasonable